qa/tasks/mgr/dashboard/test_rbd.py

```diff
@@ -869,7 +869,19 @@ def _get_config_by_name(conf_name):
         self.assertEqual(clone_format_version, 2)
         self.assertStatus(200)
 
+        # will take no effect
```

Comment:
nit: this comment is at least unclear to me if I want to follow. Can we improve ?